README.md

```diff
@@ -8,7 +8,11 @@
 There are two modules for handling SRT streaming.
 
 #### Relay
-Hwangsae Relay performs the SRT relay. Edge devices send one SRT streaming to Relay which then can be consumed by N different SRT sinks.
+Hwangsae Relay performs the SRT relay. Edge devices send one SRT streaming to Relay which then can be consumed by N different SRT sinks. It uses:
```

Comment:
imo "SRT stream" would be better than "SRT streaming"

Reply:
Sure.